client/js/shout.js

```diff
@@ -314,6 +314,11 @@ $(function() {
 		}
 	});
 
+	socket.on("topic", function(data) {
+		// .text() escapes HTML (but not quotes)
```

Comment:
> `(but not quotes)`

How is that relevant to us here? We need to escape HTML to avoid XSS, but it seems to me that escaping quote would be only relevant to avoid SQL injection, which is not an issue here.